sfge/src/test/java/com/salesforce/graph/ops/TraversalUtilTest.java

```diff
@@ -63,6 +68,47 @@ public class TraversalUtilTest {
                     + "	}\n"
                     + "}\n";
 
+    private String[] interfaceTestSourceCodes = {
```

Comment:
Added aggressive test coverage for the new traversal, with every test case I could think of.